src/api/h5pApi.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import fetch from 'isomorphic-fetch';
+import { h5pHost } from '../config.js';
+
+import { resolveJsonOrRejectWithError } from '../utils/apiHelpers';
+
+const getHeaders = () => ({
+  headers: {
+    'content-type': 'Content-Type: application/json',
+  },
+});
+
+export const fetchH5p = async id => {
```

Comment:
Bruke et anna navn for å tydliggjøre kva som skjer?
```suggestion
export const fetchH5pLicenseInformation = async id => {
```